src/legacy/ui/public/saved_objects/__tests__/saved_object.js

```diff
@@ -110,66 +110,6 @@ describe('Saved Object', function () {
         sinon.stub(esDataStub, 'create').returns(Bluebird.reject(mock409FetchError));
       }
 
-      describe('when true', function () {
```

Comment:
Why were these tests removed?

Reply:
They are covering the confirmation of an overwrite. this was done by stubbing the confirmModalPromise decorator of Angular. Since this is now deangularized, it no longer works, but I'm preparing a follow up PR migrating the tests to Jest. Furthermore, there is a older comment in the code 
```
  // TODO: make onTitleDuplicate a required prop and remove UI components from this class
 ```
so the duplicate detection should no longer be done in the way the test does. Ideally this can be cleanup up in a follow up PR